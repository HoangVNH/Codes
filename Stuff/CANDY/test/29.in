10
22
XXXXXDDXXVVVVVXXXXVVVV
23
DXXDDDDVVVVVVVVVVDDDDDD
28
DDDXXXXXDDVVXXXXXXXXDVVDDDDD
48
XXXVXXXVVDDDDDDVVVDDDDDVVDDDDDVVVDXXXXVVVVVXXXVV
21
VXXDDDDXXXXVVVVVVVVDD
42
DDDXXXXXDDDXXXXVVVVVVXXXXXVVVVVVVVVVVVVVVV
42
VVVDDDDDDDDVVVVVVVVXXXXXDDDDDVVVDDDDDVVVVV
26
DDDXXXXXXVVVVVVVVVVXVDXDDD
43
VVVDDXXXXXVVVXXXDDVVVXDVVVVVVVXXXXXXXXVVVVD
37
VVVVDDDDVVVVVXXXXXXXXXXXXVVVVVVVVVVVV
