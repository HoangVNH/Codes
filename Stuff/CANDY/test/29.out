21
22
27
47
20
40
41
25
42
36
